apps/wallet/src/ui/app/shared/card-layout/index.tsx

```diff
@@ -1,66 +1,113 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import cn from 'classnames';
+import { cva, type VariantProps } from 'class-variance-authority';
 
+import { Heading } from '_app/shared/heading';
 import PageTitle from '_app/shared/page-title';
+import { Text } from '_app/shared/text';
 import Icon, { SuiIcons } from '_components/icon';
 
 import type { ReactNode } from 'react';
 
-import st from './CardLayout.module.scss';
+const cardLayoutStyles = cva(
+    [
+        'flex flex-col flex-nowrap rounded-20 items-center justify-center w-popup-width h-popup-height bg-alice-blue shadow-wallet-content',
```

Comment:
Can we remove the `w-popup-width` `h-popup-height` so we don't force card to have a specific size? I think page layout already does that

Reply:
This was mostly converted from the SCSS `max-width: sizing.$popup-width;
        max-height: sizing.$popup-height;`

Reply:
I am not sure that the new design requires changes to the existing styling of the CardLayout. (maybe some additions) So why change the existing styling? The existing style tries to stretch/shrink depending the available space IIRC. Why converting it to tailwind has to change it?

Reply:
>This was mostly converted from the SCSS max-width: sizing.$popup-width; max-height: sizing.$popup-height;

That is for when mode is box, and is max-width/height and this should move to the mode variant for box right?

Reply:
done